feat: UTC-447: Contributed projects fields in membership detail endpoint

mcanu opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/20488830051)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/714
<!-- Follow Merge -->





<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

mcanu committed:
Added new fields to detail response

mcanu committed:
Added tests

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20465852214

matt-bernstein commented on label_studio/organizations/serializers.py:
```diff
@@ -21,25 +26,37 @@ class Meta:
+@extend_schema_serializer(
+    deprecate_fields=[
+        'created_projects',
+        'contributed_to_projects',
```
why is this being deprecated if you're adding it?

mcanu replied:
these two fields already appear nested in the `user` field when listing memberships. I moved those fields to the root of the response, so we don't "contaminate" the user entity with this fields only relevant to this endpoint. The detail endpoint didn't have these, so I added them to the root directly

mcanu replied:
The deprecated fields are only the ones nested in user not the ones in the root.

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20468708273

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-utc-447-2'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20468708273

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20468782940

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/20488672393

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20488759076)

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20488728803

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-utc-447-2'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20488728803

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20488830051

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/20490283241
